app/assets/stylesheets/_card.scss

```diff
@@ -0,0 +1,53 @@
+.ArticleHeader {
+  @include display(flex);
+  @include align-items(center);
+  @include justify-content(space-between);
+  border-bottom: 1px solid $light-grey;
+}
+
+.EmptyTable {
+  @include display(flex);
+  @include align-items(center);
+  @include justify-content(center);
+  text-align: center;
+}
+
+.card {
+  border-radius: $corner-radius;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
+  background: $white;
+  border: 0;
+  margin-bottom: 25px;
+  $card-padding: 25px;
+  $thin-card-padding: 8px;
+
+  .common-top {
+    margin: 0;
+    overflow: hidden;
+  }
+
+  .top {
+    @extend .common-top;
+    padding: $card-padding;
+
+  }
+
+  .thin-top {
+    @extend .common-top;
+    padding: $thin-card-padding;
+  }
+
+  .bottom {
+    margin-top: 20px;
+
+    &.full {
+      float: none;
+      width: auto;
+      padding-bottom: 20px;
+    }
+  }
+
+  .action-form {
+    padding: $card-padding;
+  }
+}
```

Comment:
we should take a whack at using SUIT for all of these things and rejigger this SCSS file structure to use 7+1 while we're in here replacing all the SCSS. also, we should rip out bootstrap :)